app/src/main/java/tech/relaycorp/letro/account/model/Account.kt

```diff
@@ -27,6 +27,7 @@ data class Account(
     val veraidPrivateKey: ByteArray,
     val veraidMemberBundle: ByteArray? = null,
     val awalaEndpoint: String? = null,
+    val publicThirdPartyNodeId: String? = null,
     @AccountStatus val status: Int = CREATION_WAITING,
     val token: String? = null,
```

Comment:
Is this the token you'd enter in the account linking screen? We shouldn't store it.

Reply:
No, we get this node ID from `ThirdPartyEndpoint`, after we import connection params. I use it later to check that vera id member bundle came from expected sender (this endpoint)

Reply:
Are you referring to the `token` field?

Reply:
No, I mean that we need to check if messages are coming from expected senders. Let me explain in a bit more details. 

**Problem**
1. We have a message "application/vnd.veraid.member-bundle"
2. We need to check if it came from the expected sender (public third party endpoint)

**Solution**
1. We send account linking request to receive connection params
2. We receive connection params
3. We import connection params using `PublicThirdPartyEndpoint.import(message.content)`. **It is the public third party endpoint, which has to send us vera id member bundle later.** So I store it's node id to the database to check it later
4. We send "application/vnd.veraid-authority.member-public-key-import" message with token and public key
5. We receive the "application/vnd.veraid.member-bundle" message and check if the node id equals to the node id from the step 3

Reply:
Ah, yes! Sorry, I understand now 👌🏾